src/components/panel/panel.js

```diff
@@ -1950,25 +1954,36 @@ MdPanelPosition.prototype._calculatePanelPosition = function(panelEl, position)
   var targetRight = targetBounds.right;
   var targetWidth = targetBounds.width;
 
+  function reverseXPosition (position) {
```

Comment:
Please follow the same styles in the rest of this file. Everything is using prototype. No functions are defined like this.